flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -266,6 +313,81 @@ def process(self, element, **kwargs):
         yield operation.item
 
 
+@experimental()
+class MaskDetectedDetails(PTransform):
+
+    def __init__(
+            self,
+            project=None,
+            location="global",
+            template_name=None,
+            deidentification_config=None,
+            timeout=None):
+
+        self.config = {}
+        self.project = project
+        self.timeout = timeout
+        self.location = location
+
+        if template_name is not None:
+            self.config['deidentify_template_name'] = template_name
+        else:
+            self.config['deidentify_config'] = deidentification_config
+
+    def expand(self, pcoll):
+        if self.project is None:
+            self.project = pcoll.pipeline.options.view_as(
+                GoogleCloudOptions).project
+        if self.project is None:
+            raise ValueError(
+                'GCP project name needs to be specified '
```

Comment:
this can be on a single line, no?  Not sure if there's some linter rule on max columns, but the separate seems odd.

Reply:
This is because our lint tests determine a maximum length of 79 characters, so this line break is necessary for the test to pass without fails.